OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/config/webpack.prod.js

```diff
@@ -1,33 +1,32 @@
-const webpack = require('webpack');
-const webpackMerge = require('webpack-merge');
-const commonConfig = require('./webpack.common.js');
-const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-
-module.exports = webpackMerge(commonConfig, {
-    devtool: 'source-map',
-
-    externals: {
-        'react': 'React',
-        'react-dom': 'ReactDOM'
-    },
-
-    performance: {
-        hints: "warning"
-    },
-
-    plugins: [
-        new webpack.optimize.UglifyJsPlugin({
-            sourceMap: true,
-            compress: true,
-            mangle: {
-                screw_ie8: true,
-                keep_fnames: true
-            },
-            compress: {
-                warnings: false,
-                screw_ie8: true
-            },
-            comments: false
-        })
-    ]
-});
+const webpack = require('webpack');

```

Comment:
Global comment for code files: missing copyright and license header

Reply:
I'll add the comments to files that have comments in the project.

Reply:
The latest Office generator only puts a license & copyright comment in the `index.html` file so that's the only one I listed it in, hence why it was removed.